### ISSUE TITLE
Issues by repository

### ISSUE THREAD

**Issue by apwall** [k0shk0sh/FastHub#875]
**FastHub Version: 4.0.3**  
**Android Version: 7.0 (SDK: 24)**  
**Device Information:**  
- MANUFACTURER: LGE
- BRAND: lge  
- MODEL: LG-H918  
---

 Hello,
Is it possible to view issues and filter by repository? I manage multiple project teams and this would be a huge help to me.

Thanks,
Andy

_Sent from my Lge LG-H918 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by yakov116** [comment]
@apwall Can you be a bit more specific? as to where you would like to do this?

**Comment by apwall** [comment]
On the main screen where it says Issues they were all lumped together.
However, I just realized that if I click to my company account, I was able
to click on a particular repository and see those issues.

Ideally, it would be a filter on the main screen.  But, I have to go to
Organizations-->Tap Organization --> Tap over to Repositories --> Tap
Repository-->Tap Issues

This is actually workable for me...I just didn't realize it was possible.

Thanks,
Andy
